hooks/hooks.go

```diff
@@ -18,7 +18,6 @@ const (
 	UpgradeCharm          Kind = "upgrade-charm"
 	Stop                  Kind = "stop"
 	Remove                Kind = "remove"
-	Action                Kind = "action"
```

Comment:
this is used by the uniter

Reply:
are you going to remove the code there?

Reply:
ah, came up as unused for me when i searched, but i missed the usage, will add it back